api/controllers/game/concede.js

```diff
@@ -5,14 +5,16 @@ module.exports = async function (req, res) {
     await Game.updateOne(req.session.game).set({ result: winner });
     const game = await gameService.populateGame({ gameId: req.session.game });
     await gameService.clearGame({ userId: req.session.usr });
+    let currentMatch;
     if (game.isRanked) {
-      await sails.helpers.addGameToMatch(game);
+      currentMatch = await sails.helpers.addGameToMatch(game);
     }
     // Send socket message
     const victory = {
       gameOver: true,
       winner,
       conceded: true,
+      currentMatch: currentMatch,
```

Comment:
```suggestion
      currentMatch,
```
Shorthand will work here.